Cargo.toml

```diff
@@ -33,3 +33,6 @@ members = [
 	"validator_client",
 	"account_manager",
 ]
+
+[profile.release]
+debug = true
```

Comment:
Do we want this?

Reply:
Oh yeah good spotting. That's useful for flamegraph but I don't think it should be in master. Will remove.